spec/fixtures/users.yml

```diff
@@ -9,7 +9,7 @@ alex:
 dave:
   slack_id: U3
   slack_name: dave
-
+  
```

Comment:
Remove the trailing whitespace, leave the blank line.

Reply:
Changes at:
192c98653d079a0385dd613224ef2bdfdbdbaded